snoop/models.py

```diff
@@ -69,6 +69,9 @@ class Meta:
         # TODO: constraint does not apply to container=None rows
         unique_together = ('container', 'path')
 
+    def __str__(self):
+        return str(self.path)
```

Comment:
> These will have collisions.
That's ok, `__str__` is not used anywhere important, it's just an indication of what document this is.